### ISSUE TITLE
Brew faas-cli release not setting version from tag

### ISSUE THREAD

**Issue by johnmccabe** [openfaas/faas-cli#346]
## Expected Behaviour

The release version is not being set in the brew release, it should be set to the current tag as with the get binary release. The brew formula build step was never updated following the merge of #104.

**No action is required on this repo, however the brew release should be checked on next release**

## Current Behaviour

```
$ faas-cli version
  ___                   _____           ____
 / _ \ _ __   ___ _ __ |  ___|_ _  __ _/ ___|
| | | | '_ \ / _ \ '_ \| |_ / _` |/ _` \___ \
| |_| | |_) |  __/ | | |  _| (_| | (_| |___) |
 \___/| .__/ \___|_| |_|_|  \__,_|\__,_|____/
      |_|
Commit: 7997a467e0d1a49eae5417b8190ae0f5f5e87bac
Version: dev
```

## Possible Solution

A PR with a fix has been raised Homebrew/homebrew-core#24930.

## Steps to Reproduce (for bugs)

1. `brew install faas-cli`
2. `faas-cli version`
3. Version shows as `dev`

## Context

Need version output to be consistent across all releases.

## Your Environment

* OSX 10.13.3
* brew (up to date)

**Comment by alexellis** [comment]
Thank you for raising this John and for working on a fix.. good catch

**Comment by johnmccabe** [comment]
This has been merged upstream, recommend keeping this open until the next release.

**Comment by alexellis** [comment]
@johnmccabe I released yesterday please check brew?

**Comment by johnmccabe** [comment]
<img width="956" alt="screen shot 2018-03-13 at 10 22 32" src="https://user-images.githubusercontent.com/83862/37336170-79585d04-26a8-11e8-9662-580490e2dfd9.png">

👍 - thanks @ilovezfs

**Comment by johnmccabe** [comment]
Fixed in Homebrew/homebrew-core#25182

**Comment by alexellis** [comment]
Thanks both 🎉
